go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/go-retryablehttp v0.7.2
 	github.com/ktrysmt/go-bitbucket v0.9.48
+	github.com/microsoft/azure-devops-go-api/azuredevops/v6 v6.0.1
```

Comment:
latest release is v7  https://github.com/microsoft/azure-devops-go-api/releases/tag/azuredevops%2Fv7.1.0

Reply:
we can't use V7 until the fix `ContinuationToken` field type. It is unconsistent depending on which struct you use.

Reply:
@souleb  is it related to this issue? https://github.com/microsoft/azure-devops-go-api/issues/97 did you find it in a different struct?

Reply:
i've mentioned this to the relevant team and will let you know when it gets resolved